src/main/java/io/personium/core/PersoniumUnitConfig.java

```diff
@@ -538,13 +551,38 @@ private synchronized void doReload() {
                 this.propsOverride.setProperty((String) entry.getKey(), (String) entry.getValue());
             }
         }
+        Map<String, String> env = System.getenv();
```

Comment:
Environment variable's priority is high to switch enviroment.

Current:
1. system property
2. config file
3. environment variable
4. default value

Ideal:
1. system property
2. environment variable
3. config file
4. default value

Reference: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config